wordpress-mobile/gutenberg-mobile
# List Block fixes for iOS.

Fixes #886

This PR implements [changes on Aztec-iOS](https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1165) that fix an issue on the block list.
There is a related [gutenberg side PR](https://github.com/WordPress/gutenberg/pull/14948) needed to fix an issue pressing enter on List blocks.
Issue: 
![ios-list-end-issue](https://user-images.githubusercontent.com/9772967/56039915-98d11580-5d35-11e9-8a54-9c4545df97bf.gif)

After the fix:
![list_block](https://user-images.githubusercontent.com/9772967/56039926-9ec6f680-5d35-11e9-9b56-9c0b84ec7f60.gif)

To test:

- Create a list block with the following content:
```

<ul><li>line 1<ul><li>line 21</li><li>line 22<ul><li>line 31</li><li>line 32</li></ul></li></ul></li><li>line 3</li><li>line 4</li><li>line 5</li></ul>

```
- Add a new list item.
- Check that the new list item is properly placed.
